hr_employee_relative/tests/test_hr_employee_relatives.py

```diff
@@ -0,0 +1,29 @@
+# Copyright (C) 2018 Brainbean Apps
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo.tests import common
+
+from datetime import datetime
+from dateutil.relativedelta import relativedelta
+
+
+class TestHrEmployeeRelatives(common.TransactionCase):
+
+    def setUp(self):
+        super().setUp()
+        self.Employee = self.env['hr.employee']
+        self.EmployeeRelative = self.env['hr.employee.relative']
+
+    def test_age_calculation(self):
+        employee = self.Employee.create({
+            'name': 'Employee',
+            'relatives': [
+                (0, 0, {
+                    'relation': 'other',
+                    'name': 'Relative',
+                    'date_of_birth': datetime.now() + relativedelta(years=-42),
+                })
+            ],
+        })
+        relative = self.EmployeeRelative.browse(employee.relatives[0].id)
+        self.assertEqual(int(relative.age), 42)
```

Comment:
@alexey-pelykh here static age, next year it's changed, IMO here change like this
`age = relativedelta(datetime.now(), relative.date_of_birth)`
`record.age = age.years + (age.months / 12)`

`assertEqual(relative.age, age)`

Reply:
Not sure what you mean since 42 years are added to the test data initially, thus resulting the same. And copying code from the tested method itself has to meaning to me

Reply:
> thus resulting the same.

Yes @alexey-pelykh, Because this yesr `age` is same but next year different or increasing one.
So you can use Compute code here and check assertEqual.

Reply:
Still don't get it, every year age will be 42 due to `'date_of_birth': datetime.now() + relativedelta(years=-42),`

Reply:
@alexey-pelykh Could you please check this link https://github.com/OCA/hr/pull/458#discussion_r221875110

here Calculation are Done by me.

Reply:
Still not applicable. During creation of test data the relative is given _exactly_ 42 years of age by subtracting 42 from current date:
```
'date_of_birth': datetime.now() + relativedelta(years=-42),
```
Couple of milliseconds later it's checked if calculation returns 42, effectively checking if `relative.age` is computed as 42. Checking if `relative.age` equals to anything but constant dismisses the meaning of the test. The only change that would make sense is to put 42 in a separate variable to avoid it being a magic constant twice, yet for a such small piece of code it's not that relevant.